lib/active_model/entity/serializers/json.rb

```diff
@@ -17,16 +36,33 @@ def fetch_field_value(object_or_hash, name)
             end
           end
 
-          def represent(object_or_hash, camelize: true)
+          def represent(object_or_hash, options = {})
+            entity_options = default_represent_options.merge(options)
+
+            camelize = entity_options[:camelize]
```

Comment:
nit: it would be easier to do `camelize = entity_options.fetch(:camelize, true)`